pkg/util/mount/mount_windows.go

```diff
@@ -74,8 +74,23 @@ func (mounter *Mounter) Mount(source string, target string, fstype string, optio
 			return nil
 		}
 
-		// empty implementation for mounting azure file
-		return os.MkdirAll(target, 0755)
+		cmd := fmt.Sprintf(`$User = "AZURE\%s";$PWord = ConvertTo-SecureString -String "%s" -AsPlainText -Force;`+
```

Comment:
is `User="AZURE\%s` too restrictive? what if the node is not in azure cloud.

Reply:
good point, I will put User="AZURE\%s" into azure_file.go which would be more reasonable